test/e2e/util.go

```diff
@@ -165,6 +167,123 @@ func createRBDImage(namespace, poolName, imageName string, siz uint) error {
 	return nil
 }
 
+func createRBDCloneImage(namespace, poolName, snapName, cloneName string) error {
+	_, stderr, err := kubectl(
+		"-n", namespace, "exec", "deploy/rook-ceph-tools", "--",
+		"rbd", "clone",
+		"--rbd-default-clone-format", "2",
+		"--image-feature", "layering,deep-flatten",
+		poolName+"/"+snapName, poolName+"/"+cloneName)
+	if err != nil {
+		return fmt.Errorf("rbd clone failed. stderr: %s, err: %w", string(stderr), err)
+	}
+	return nil
+}
+
+func removeRBDImage(namespace, poolName, imageName string) error {
+	_, stderr, err := kubectl(
+		"-n", namespace, "exec", "deploy/rook-ceph-tools", "--",
+		"rbd", "rm", poolName+"/"+imageName)
+	if err != nil {
+		return fmt.Errorf("rbd rm failed. stderr: %s, err: %w", string(stderr), err)
+	}
+	return nil
+}
+
+func createRBDSnap(namespace, poolName, imageName, snapName string) error {
+	_, stderr, err := kubectl(
+		"-n", namespace, "exec", "deploy/rook-ceph-tools", "--",
+		"rbd", "snap", "create", poolName+"/"+imageName+"@"+snapName)
+	if err != nil {
+		return fmt.Errorf("rbd snap create failed. stderr: %s, err: %w", string(stderr), err)
+	}
+	return nil
+}
+
+func removeRBDSnap(namespace, poolName, imageName, snapName string) error {
+	_, stderr, err := kubectl(
+		"-n", namespace, "exec", "deploy/rook-ceph-tools", "--",
+		"rbd", "snap", "rm", poolName+"/"+imageName+"@"+snapName)
+	if err != nil {
+		return fmt.Errorf("rbd snap rm failed. stderr: %s, err: %w", string(stderr), err)
+	}
+	return nil
+}
+
+func removeAllRBDImageAndSnap(namespace, pool string) error {
+	targets, err := getImageAndSnapNames(namespace, pool)
+	if err != nil {
+		return err
+	}
+
+	images := []string{}
+	clones := []string{}
+	snaps := []string{}
+	// remove RBD snapshots first
```

Comment:
It's better to mention RBD clone images.

```suggestion
	// remove RBD clone images of targets, RBD anspshots of targets, and target images in order
	// to remove them cleanly. 
```